project/Smithy4sBuildPlugin.scala

```diff
@@ -390,6 +391,20 @@ object Smithy4sBuildPlugin extends AutoPlugin {
   }
 
   lazy val jsDimSettings = simpleJSLayout ++ Seq(
+    scalacOptions ++= {
+      // Map the sourcemaps to github paths instead of local directories
+      val flag =
+        if (scalaVersion.value.startsWith("3")) "-scalajs-mapSourceURI:"
+        else "-P:scalajs:mapSourceURI:"
+      val localSourcesPath = baseDirectory.value.toURI
+      val headCommit = git.gitHeadCommit.value.get
+      scmInfo.value.map { info =>
+        val remoteSourcesPath =
+          s"${info.browseUrl.toString
+            .replace("github.com", "raw.githubusercontent.com")}/$headCommit"
+        s"${flag}:$localSourcesPath->$remoteSourcesPath"
```

Comment:
you have an extra `:` here (you already have one in the flag)

Reply:
I removed `:` from the `flag` and have tests passing locally

Reply:
What a silly mistake :P, thank you!